test/expected/build/library.e/build-manifest/resources/library/e/library-preload.js

```diff
@@ -0,0 +1,21 @@
+//@ui5-bundle library/e/library-preload.js
+sap.ui.require.preload({
+	"library/e/library.js":function(){
+/*!
+ * UI development toolkit for HTML5 (OpenUI5)
+ * (c) Copyright 2009-xxx SAP SE or an SAP affiliate company.
```

Comment:
is it expected to not have the end year replaced in the copyright?

Reply:
Yeah. The copyright configuration for this project does not use the `${currentYear}` configuration. I guess this is so we don't have to mock `Date` to get a stable year here.